src/extension/sdk/dev_tools/embedded_view.ts

```diff
@@ -25,6 +26,9 @@ window.addEventListener('message', (event) => {
 			// https://github.com/flutter/devtools/issues/2775
 			window.dispatchEvent(new KeyboardEvent('keydown', message.data));
 			break;
+		case "launchUrl":
+			vscode.postMessage({command: 'launchUrl', data: message.data});
```

Comment:
I think we should change this tso the URL is included inside ` data` object instead of directly. That way if we need to ever pass additional data with the URL in future (for example whether it should open in another embedded window vs external, or something), it would be non-breaking to change it (and not require coordination with a DevTools change).

Reply:
Done